ts/src/binance.ts

```diff
@@ -8100,6 +8098,8 @@ export default class binance extends Exchange {
             request['network'] = network;
             params = this.omit (params, 'network');
         }
+        const precisionAmount = this.currencyToPrecision (code, amount, network);
```

Comment:
`network` is overrided above, could you please verify if we should put network after this line `network = this.safeString (networks, network, network); // handle ERC20>ETH alias` of before?

I think that it might not work since the final `network` value is the external network, not the one used by ccxt

Reply:
We did not use this mapping when fetching currencies, I moved the line before and also added the mapping logic in `fetchCurrencies`

Reply:
Alright